services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/pdf/KaivuilmoitusPdfEncoder.kt

```diff
@@ -150,7 +118,8 @@ object KaivuilmoitusPdfEncoder {
         builder.append("Pinta-ala: ${totalArea.format()} m²\n")
         builder.append("Katuosoite: ${alue.katuosoite}\n")
         builder.append(
-            "Työn tarkoitus: ${alue.tyonTarkoitukset.map { it.format() }.joinToString ( ", " )}\n")
+            "Työn tarkoitus: ${alue.tyonTarkoitukset.map { it.format() }.joinToString ( ", " )}\n"
```

Comment:
This could be written as `"Työn tarkoitus: ${alue.tyonTarkoitukset.joinToString(", ") { it.format() }}\n"`